rollingpin/main.py

```diff
@@ -159,6 +166,10 @@ def _main(reactor, *raw_args):
     if config["graphite"]["endpoint"]:
         enable_graphite_notifications(config, event_bus, args.components)
 
+    if config["elasticsearch"]["endpoint"]:
+        enable_elastic_search_notifications(config, event_bus, args.components)
+
+
```

Comment:
nit: only one blank within functions. you might try running `pep8` on the diff to see if any other style nits are being introduced